### PR TITLE
bump image version and apiVersion

### PR DESCRIPTION
**Pull request by zouyee**
**What this PR does / why we need it**:
bump image version and apiVersion

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```NONE

```

### CODE REVIEW

File: cluster/addons/metadata-proxy/gce/metadata-proxy.yaml [hunk 1]
```diff
@@ -38,7 +38,7 @@ spec:
       dnsPolicy: Default
       containers:
       - name: metadata-proxy
-        image: gcr.io/google_containers/metadata-proxy:v0.1.6
+        image: gcr.io/google_containers/metadata-proxy:v0.1.7
```
Comment (roberthbailey) [inline cluster/addons/metadata-proxy/gce/metadata-proxy.yaml]: @ihmccreery - what changed in this version?

Comment (zouyee) [reply cluster/addons/metadata-proxy/gce/metadata-proxy.yaml]: https://github.com/GoogleCloudPlatform/k8s-metadata-proxy/pull/13

and change base image.

**Comment by roberthbailey** [comment]
It looks like you meant to close this PR, so I'm going to click the button. If you would still like it to be reviewed / merged, please reopen it.

**Pull request closed without merging**